modules/migration-mtc-release-notes-1-7.adoc

```diff
@@ -0,0 +1,34 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+
+[id="migration-mtc-release-notes-1-7_{context}"]
+= {mtc-full} 1.7 release notes
+
+[id="new-features-and-enhancements-1-7_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* The {mtc-full} ({mtc-short}) operator now depends upon the OpenShift API for Data Protection (OADP) operator. When you install the {mtc-short} operator, the Operator Lifecycle Manager (OLM) automatically installs the OADP operator in the same namespace.
+
+* You can migrate from a source cluster that is behind a firewall to a cloud-based destination cluster by establishing a network tunnel between the two clusters using the `crane tunnel-api` command.
+
+* Converting storage classes in the MTC web console: You can convert the storage class of a persistent volume (PV) by migrating it within the same cluster.
+
+[id="deprecated-features-1-7_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
+* {mtc-short} version 1.5 and 1.6 are deprecated. Support for them will be removed in a future version.
```

Comment:
1.5 and 1.6 are not deprecated unless there is some formal definition for this that I'm unaware of. We intend to fully support this until 1.8 is released when support will be dropped.

Reply:
+1